west.yml

```diff
@@ -37,7 +37,7 @@ manifest:
       revision: a9d9dc78760a7751576987401fde7263eaa6c094
     - name: fw-nrfconnect-mcuboot
       path: mcuboot
-      revision: 9d9a463f68fea23a0432e8dd597b7baad4a659ed
+      revision: d75212468ce0f88ed57a4aacac8aa07cc8a725bc
```

Comment:
From what I [learned](https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/453#discussion_r255080686), instead of a revision hash you should specify:
`revision: pull/11/head`
so that CI is run against that PR.
After the upmerge, you should edit this PR again to include the SHA.

Reply:
Thanks for the tip.